cordformation/src/main/kotlin/net/corda/plugins/Dockerform.kt

```diff
@@ -60,9 +60,8 @@ open class Dockerform @Inject constructor(objects: ObjectFactory) : Baseform(obj
     var dockerConfig: Map<String, Any> = emptyMap()
 
     /**
-     * Docker image for this node, or null if one hasn't been specified.
+     * Docker image for this node
      */
-    @get:Optional
     @get:Input
     var dockerImage: String? = null
```

Comment:
The `dockerImage` property should still never be `null` though. Maybe use a "lazy" Gradle property here?
```kotlin
@get:Input
val dockerImage: Property<String> = objects.property(String::class.java)
```
And then you wouldn't need to change the types of your `MutableMap<K,V>` objects either.